kwanj-k/storemanager-API
# #161199177 Add a get products endpoint

### What does this PR do?
Adds a get products or one product endpoint

#### Description of Task to be Completed ?
-- Have a working get products endpoint
-- Consider different edge cases when getting products
-- Get the get products tests to pass

### How should this be manually tested?
* After cloning the repo,git pull the branch ft-get-products- #161199177
* Install a virtual enviroment and activate it [Refer here](https://docs.python-guide.org/dev/virtualenvs/)
* Pip install the the requirements.txt
* Run the following:
  ``` export FLASK_APP='run.py' ```
  ``` export APP_SETTINGS='development' ```
   ``` pytest --cov-report term-missing --cov=app ```

 All related products tests should pass

### What are the relevant Pivotal Tracker stories?
  #161199177
